skietbaan_fe/src/components/Login.js

```diff
@@ -1,41 +1,54 @@
 import React, { Component } from 'react';
 import { Container, Col,FormGroup, Label, Input, Button, Form} from 'reactstrap';
-import '../components/LoginStyles.css';
-import { validatePassword, validateEmail, validateUsername } from './Validators.js';
+import '../components/RegisterStyles.css';
+import {  validateUsername } from './Validators.js';
 import { getCookie } from './cookie.js';
 
 class App extends Component {
   constructor(props) {
     super(props);
     this.state = {
       usernameValue: "",
-      emailValue: "",
       passwordValue: "",
       validForm: false,
       tokenValue : "",
     }
     this.Login = this.Login.bind(this);
     this.handleChange = this.handleChange.bind(this);
+    this.Validate = this.Validate.bind(this);
 
   }
   handleChange({ target }) {
     this.setState({
       [target.name]: target.value,
     });
+    let isValid = true;
```

Comment:
I don't think isValid is set anywhere else in this scope. Might as well make this a const.